packages/dapp/components/lib/hooks/contracts/use-protocol-contracts.ts

```diff
@@ -27,6 +27,10 @@ import StakingFacetArtifact from "@ubiquity/contracts/out/StakingFacet.sol/Staki
 import StakingFormulasFacetArtifact from "@ubiquity/contracts/out/StakingFormulasFacet.sol/StakingFormulasFacet.json";
 import TWAPOracleDollar3poolFacetArtifact from "@ubiquity/contracts/out/TWAPOracleDollar3poolFacet.sol/TWAPOracleDollar3poolFacet.json";
 import UbiquityPoolFacetArtifact from "@ubiquity/contracts/out/UbiquityPoolFacet.sol/UbiquityPoolFacet.json";
+// other related contracts
+import SushiSwapPoolArtifact from "@ubiquity/contracts/out/SushiSwapPool.sol/SushiSwapPool.json";
+import IMetaPoolArtifact from "@ubiquity/contracts/out/IMetaPool.sol/IMetaPool.json";
+import UniswapV2PairABI from "@/components/config/abis/uniswap-v-2-pair.json";
```

Comment:
fix this naming, to something camelCase, this is not allowed, `uniswapV2Pair.json`

Reply:
> fix this naming, to something camelCase, this is not allowed, `uniswapV2Pair.json`

Yeah, I didn't pay attention to it 🤦‍♂️